github/examples_test.go

```diff
@@ -101,3 +101,39 @@ func ExamplePullRequestsService_Create() {
 
 	fmt.Printf("PR created: %s\n", pr.GetHTMLURL())
 }
+
+func ExampleOrganizationsService_ListTeams() {
+	// This example shows how to get a team ID corresponding to a given team name.
+
+	// Note that authentication is needed here as you are performing a lookup on
+	// an organization's administrative configuration, so you will need to modify
+	// the example to provide an oauth client to github.NewClient() instead of nil.
+	// See the following documentation for more information on how to authenticate
+	// with the client:
+	// https://godoc.org/github.com/google/go-github/github#hdr-Authentication
+	client := github.NewClient(nil)
+
+	teamName := "Developers team"
+	ctx := context.Background()
+	opts := &github.ListOptions{}
+
+	for {
+		teams, resp, err := client.Organizations.ListTeams(ctx, "myOrganization", opts)
+		if err != nil {
+			fmt.Println(err)
+			return
+		}
+		for _, t := range teams {
+			if t.GetName() == teamName {
+				fmt.Printf("Team %q has ID %d\n", teamName, t.GetID())
+				return
+			}
+		}
+		if opts.Page == resp.LastPage {
+			break
+		}
```

Comment:
There are many valid ways of doing pagination and detecting last page. This is one of them.

But I think we should be consistent with the example provided at https://godoc.org/github.com/google/go-github/github#hdr-Pagination, and use the same style here.

Reply:
That totally makes sense.